rpcserver.go

```diff
@@ -2690,6 +2682,12 @@ func (r *rpcServer) CloseChannel(in *lnrpc.CloseChannelRequest,
 			maxFee, deliveryScript,
 		)
 	}
+
+	// If the user doesn't want to wait for the txid to come back then we
+	// will exit early.
+	if in.NoWait {
+		return nil
```

Comment:
The idea is to NOT send updates on the closing client if they specify no_wait. Can you clarify what you mean here?

Reply:
yeah i don't think that's a good idea tho - I think the callers would want to know the closing status of a given channel, which is already async as they just listen to the updates sent from the closing client. If not, they can just bail out. Then this `no_wait` flag enhances the experience of closing channels, after user calls `closechannel`, it  returns immediately. 
```go
// By using the flag, it can now return immediately.
closeClient := alice.RPC.CloseChannel(...)

// User can then still wait for the closing result.
ht.ReceiveCloseChannelUpdate(closeClient)

// Or decide to not wait and continue.
...
```

Reply:
I've adjusted this to send an empty update instantly to allow the CLI and REST interfaces to fast return.